examples/testing/karma.conf.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function(config) {
+  config.set({
+    plugins: ['karma-json-result-reporter'],
+    reporters: ['dots', 'progress', 'json-result'],
+    logLevel: config.LOG_DEBUG,
+    colors: false,
+    jsonResultReporter: {
+      outputFile: "${TEST_UNDECLARED_OUTPUTS_DIR}/karma-result.json",
```

Comment:
Is there a reason for the deferred variable substitution? If so, can you please add an explanation here?
Since this file is executed under Bazel (it is require'd in the same context as the base config file), it should have access to the environment variables and they should be used directly.

Reply:
Right. That's not necessary. Thanks!